edx_sga/tasks.py

```diff
@@ -61,10 +61,15 @@ def _compress_student_submissions(zip_file_path, block_id, course_id, locator):
     student_submissions = _get_student_submissions(block_id, course_id, locator)
     if not student_submissions:
         return
+
+    log.info("Fetched all submission count: %d for zip path: %s ", len(student_submissions), zip_file_path)
     # Build the zip file in memory
     zip_file_bytes = BytesIO()
-    with zipfile.ZipFile(zip_file_bytes, 'w') as zip_pointer:
+    with zipfile.ZipFile(zip_file_bytes, 'w', compression=zipfile.ZIP_DEFLATED, allowZip64=True) as zip_pointer:
```

Comment:
I don't think we want `allowZip64` here. I don't think we will need to handle zip files larger than 2gb after compression, and using this option makes the zip files incompatible with certain command line tools

Reply:
Adding the `compression` flag is a very good idea though